scripts/views/eventView.js

```diff
@@ -9,12 +9,41 @@
   eventHandler.lookup = function (e) {
     e.preventDefault();
     var zip = $('#look-up input').val();
+    if (zip && zip.length === 5) {
+      repZip = zip;
+    } else if (zip && zip.length > 5) {
+      zipFive = zip.match(/\d{5}/g);
```

Comment:
Can we declare these with var (or let if we have a fallback for ES5)?  Would prefer to keep them with restricted scope.